src/bootstrap.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
```

Comment:
This file is simply a paste of the core application initialization code that used to be in `index.js`. It was created to be shared between `index.js` for auto init, and `init.js` for manual init.